src/components/header/header_alert/header_alert.tsx

```diff
@@ -22,36 +26,27 @@ export const EuiHeaderAlert: FunctionComponent<EuiHeaderAlertProps> = ({
   date,
   text,
   title,
+  badge,
   ...rest
 }) => {
   const classes = classNames('euiHeaderAlert', className);
 
   return (
-    <EuiI18n token="euiHeaderAlert.dismiss" default="Dismiss">
-      {(dismiss: string) => (
-        <div className={classes} {...rest}>
-          <EuiButtonIcon
-            aria-label={dismiss}
-            iconType="cross"
-            size="s"
-            className="euiHeaderAlert__dismiss"
-          />
-
-          <div className="euiHeaderAlert__title">{title}</div>
-
-          <div className="euiHeaderAlert__text">{text}</div>
-
-          <EuiFlexGroup justifyContent="spaceBetween">
-            <EuiFlexItem grow={false}>
-              <div className="euiHeaderAlert__action euiLink">{action}</div>
-            </EuiFlexItem>
-
-            <EuiFlexItem grow={false}>
-              <div className="euiHeaderAlert__date">{date}</div>
-            </EuiFlexItem>
-          </EuiFlexGroup>
-        </div>
+    <div className={classes} {...rest}>
+      <EuiFlexGroup justifyContent="spaceBetween">
+        <EuiFlexItem>
+          <div className="euiHeaderAlert__date">{date}</div>
```

Comment:
Can we replace the `div` with a [`time`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time) tag? 
```suggestion
          <time className="euiHeaderAlert__date">{date}</time>
```

What might be a bit tougher but would love to see as well is to support automatically filling in a `datetime` attribute. I'm not sure what `EuiI18n` supports or not but instead of relying on Kibana (or other implementing devs) to pass in internationalized date strings, if EUI could accept an [ISO8601](https://en.wikipedia.org/wiki/ISO_8601) and then use that both in the `datetime` attribute and then "translate" it for the human readable section that'd be awesome. Would look something like:
```jsx
<time className="euiHeaderAlert__date" datatime={date}>
	// NOTE: this is pseudo-code, I don't actually know how to make EuiI18n take a date 
	<EuiI18n value={date} /> 
</time>
```

Reply:
@myasonik For the time being I'll just replace `div` with `time`. Sound good?

Reply:
It's ok if `EuiI18n` can't handle dates...

Reply:
Since `{date}` is a passed prop from the consuming app, EUI doesn't need to do any localization on it. It should be on the consumer to do that level. We only localize strings in EUI if they are hard-coded.